### DISCUSSION TITLE
how to get started?

### DISCUSSION THREAD

**Discussion by ghost** [github/india#45] [Open Source] [answered]
How to get started or grow the first open source project of our own?

how to get the suggestions from the community?

Please share tips and tricks

**Comment by liyasthomas** [comment, marked as answer]
- Launch your product as soon as possible
- Ask feedbacks from your friends and colleges.
- Iterate over your project to have essential set of features
- Promote your project very often
- Find places where your potential users are hanging out - for softwares it's mostly developer forums like dev.to, hashnode.com, sub-reddits etc.
- Share your progress on social media platforms
- Ask for help and feedbacks very often
- Participate and share your projects in events like Hacktoberfest - let your users know the project is contributor friendly.
- Be consistent - respond to issues, PRs, start a community in public platforms like Discord, Telegram, Slack etc.

**Comment by tsadarsh** [comment]
You may want to see this: https://guides.github.com/activities/hello-world/

**Comment by richakumar** [comment]
Also take a look at https://github.com/open-source

**Comment by pranavq212** [comment]
+1 to link shared by @richakumar, that is quite useful for start.

You can also look at course from Linux foundation **[Fundamentals of Professional Open Source Management (LFC210)](https://training.linuxfoundation.org/training/fundamentals-of-professional-open-source-management/)** for non tech concepts for products that heavily rely on open source & are enterprise level development.

Completing this will give you overview of the key phases of developing a Professional Open Source Management program, which includes Open Source Software and Open Source Management Basics, Open Source Management Strategy, Open Source Policy, Open Source Processes, and Open Source Management Program Implementation.

Also there is badge for completion at acclaim like [this](https://www.youracclaim.com/badges/f77f2fd7-980d-4256-9ec2-10747a9f19cd)

Also there are many learning path on https://lab.github.com/ with step-to-step hands-on instruction for learning while doing on github, some very basic ones are:

- [First Day on GitHub](https://lab.github.com/githubtraining/first-day-on-github)
- [First Week on GitHub](https://lab.github.com/githubtraining/first-week-on-github)
- [Community Starter Kit](https://lab.github.com/githubtraining/community-starter-kit)
